#service for gettin airdata from environment
#send the states of model, and get airdata

last_letter_2/model_states states
---
last_letter_2/air_data airdata